scripts/dev.ts

```diff
@@ -0,0 +1,542 @@
+      },
+      {
+        type: 'guild-createProposal',
+        from: '0x79706c8e413cdaee9e63f282507287b9ea9c0928',
+        data: {
+          guildName: 'DXDGuild',
+          to: ['DXDGuild'],
+          callData: [
+            new web3.eth.Contract(ERC20Guild.abi).methods
+              .setPermission(
+                [ZERO_ADDRESS],
+                [ANY_ADDRESS],
+                [ANY_FUNC_SIGNATURE],
+                [web3.utils.toWei('5').toString()],
+                [true]
+              )
+              .encodeABI(),
+          ],
+          value: ['0'],
+          totalActions: '1',
+          title: 'Proposal Test #0',
+          description:
+            'Allow call any address and function and send a max of 5 ETH per proposal',
+        },
+      },
+      {
+        type: 'guild-voteProposal',
+        from: '0xc73480525e9d1198d448ece4a01daea851f72a9d',
+        data: {
+          guildName: 'DXDGuild',
+          proposal: 0,
+          action: '1',
+          votingPower: web3.utils.toWei('90').toString(),
+        },
+      },
+      {
+        time: moment.duration(10, 'minutes').asSeconds(),
+        type: 'guild-endProposal',
+        from: '0xc73480525e9d1198d448ece4a01daea851f72a9d',
+        data: {
+          guildName: 'DXDGuild',
+          proposal: 0,
+        },
+      },
+    ],
+  };
+
+  const { networkContracts, addresses } = await hre.run('deploy-dxvote', {
+    deployconfig: JSON.stringify(deployconfig),
+  });
+
+  const developConfig = {
+    cache: {
+      fromBlock: 0,
+      toBlock: 1,
+      ipfsHash:
+        '0x0000000000000000000000000000000000000000000000000000000000000000',
+    },
+    contracts: networkContracts,
+    recommendedCalls: [],
+    proposalTemplates: [],
+    proposalTypes: [
+      {
+        id: 'contributor',
+        title: 'Contributor',
+        scheme: 'QuickWalletScheme',
+      },
+      {
+        id: 'custom',
+        title: 'Custom',
+      },
+    ],
+    contributionLevels: [
+      {
+        id: '1',
+        dxd: 2000,
+        stable: 4000,
+        rep: 0.1667,
+      },
+      {
+        id: '2',
+        dxd: 3000,
+        stable: 5000,
+        rep: 0.1667,
+      },
+      {
+        id: '3',
+        dxd: 4000,
+        stable: 6000,
+        rep: 0.1667,
+      },
+      {
+        id: '4',
+        dxd: 5000,
+        stable: 7000,
+        rep: 0.1667,
+      },
+      {
+        id: '5',
+        dxd: 6000,
+        stable: 8000,
+        rep: 0.1667,
+      },
+    ],
+    tokens: [
+      {
+        address: addresses.DXD,
+        name: 'DXdao on Localhost',
+        decimals: 18,
+        symbol: 'DXD',
+        fetchPrice: true,
+        logoURI:
+          'https://s2.coinmarketcap.com/static/img/coins/200x200/5589.png',
+      },
+    ],
+    guilds: [addresses.DXDGuild, addresses.REPGuild],
+  };
+
+  await writeFileSync(
+    './src/configs/localhost/config.json',
+    JSON.stringify(developConfig, null, 2)
+  );

```

Comment:
We can be sure that the folder exists, if not this will fail. 
like:
```
  mkdirSync(path.resolve(__dirname, '../src/configs/localhost'), {
    recursive: true,
  });
  await writeFileSync(
    path.resolve(__dirname, '../src/configs/localhost/config.json'),
    JSON.stringify(developConfig, null, 2)
  );
```

Reply:
done in https://github.com/DXgovernance/dxvote/pull/687/commits/b0cca80c4b9fc3a521958523e6e6bcc46749023d